testing/integration_test.go

```diff
@@ -34,11 +34,12 @@ import (
 )
 
 var (
-	repo   = flag.String("repo", "https://github.com/GoogleCloudPlatform/cloud-profiler-nodejs.git", "git repo to test")
-	branch = flag.String("branch", "", "git branch to test")
-	commit = flag.String("commit", "", "git commit to test")
-	pr     = flag.Int("pr", 0, "git pull request to test")
-	runID  = strings.Replace(time.Now().Format("2006-01-02-15-04-05.000000-0700"), ".", "-", -1)
+	repo        = flag.String("repo", "https://github.com/GoogleCloudPlatform/cloud-profiler-nodejs.git", "git repo to test")
+	branch      = flag.String("branch", "", "git branch to test")
+	commit      = flag.String("commit", "", "git commit to test")
+	pr          = flag.Int("pr", 0, "git pull request to test")
+	runID       = strings.Replace(time.Now().Format("2006-01-02-15-04-05.000000-0700"), ".", "-", -1)
+	runV8Canary = flag.Bool("v8_canary", false, "if tests should be run with the v8 canary build")
```

Comment:
The name of the flag and its description plus the name of the flag variable are a bit confusing - it's as if setting this flag will make _all_ tests run using the V8 canary, while what setting this flag does is adding the V8 canary test case to the list of existing test cases. A better name would be along the lines of making it clear that this is a bonus test, not replacement. Maybe something like `-enable_v8_canary_test`.

Also, it's better to name the flag variable using same words as the flag. Here it is `runV8Canary` vs. `v8_canary` which adds confusion.

Reply:
Similarly, update the PR title.